<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFesTesterClass</class>
 <widget class="QMainWindow" name="CFesTesterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>421</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>BaseInfo</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>project path:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_open">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0" rowspan="2" colspan="2">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Func</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="m_RadioDo">
            <property name="text">
             <string>DO</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="m_RadioAo">
            <property name="text">
             <string>AO</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="m_RadioMsg">
            <property name="text">
             <string>Msg</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>DO</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Channel</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="m_ComChannel"/>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>DOUT</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QComboBox" name="m_ComDOut"/>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Operation</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QComboBox" name="m_ComDoOper">
              <item>
               <property name="text">
                <string>空</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>选择</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>预置</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>执行</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>全套</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>AO</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Channel</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="m_AoChannelComBox"/>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>AOUT</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QComboBox" name="m_AoComBox"/>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Value</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QLineEdit" name="lineEdit_3"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>IoMsg</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Channel</string>
           </property>
          </widget>
          <widget class="QComboBox" name="m_MsgComBox"/>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Msg</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_2"/>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="m_BtnLoad">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="m_OK">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="m_Cancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="m_BtnUnLoad">
         <property name="text">
          <string>UnLoad</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>446</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="cfestester.qrc"/>
 </resources>
 <connections/>
</ui>
